Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -67,7 +69,9 @@ void doDepositionShapeN(const GetParticlePosition& GetPosition,
 #endif
 
     const amrex::Real xmin = xyzmin[0];
+#if (defined WARPX_DIM_3D)
     const amrex::Real ymin = xyzmin[1];
+#endif
```

Comment:
Fixes the warning:
`./Source/Particles/Deposition/CurrentDeposition.H:70:23`: warning: unused variable ‘ymin’ [-Wunused-variable]
     const amrex::Real ymin = xyzmin[1];